src/core/catalog.js

```diff
@@ -1533,6 +1534,38 @@ class Catalog {
           }
           break;
 
+        case "SetOCGState":
+          const state = action.get("State");
+          const preserveRB = action.get("PreserveRB");
+
+          if (!Array.isArray(state) || state.length === 0) {
+            break;
+          }
+          const stateArr = [];
+
+          for (const elem of state) {
+            if (elem instanceof Name) {
+              switch (elem.name) {
+                case "ON":
+                case "OFF":
+                case "Toggle":
```

Comment:
Does this casing originate from the PDF specification? It seems a bit odd that ON/OFF are written with capital letters only while Toggle is not.

Reply:
Yes, unfortunately this is how it's being specified; please see https://web.archive.org/web/20220309040754if_/https://www.adobe.com/content/dam/acom/en/devnet/pdf/pdfs/PDF32000_2008.pdf#G11.1990554